agents.py

```diff
@@ -967,21 +967,16 @@ def execute_action(self, agent, action):
 
         agent.bump = False
         if action == 'TurnRight':
```

Comment:
I think the if statement could be streamlined if we rewrote it as:

```
if action in ['TurnRight', 'TurnLeft', 'Forward']:
            super().execute_action(agent,action)
            agent.performance -= 1
elif action == 'Grab':
            super().execute_action(agent,action)
```

I think the above looks a bit better.

Reply:
I had that in mind, but I thought to myself that being more articulate and reasonably redundant in code would help the student retaining the information **that is my case**.

Reply:
But actually seeing it, it really didn't help much.

Reply:
We can even add 'Grab' action to the list because the performance also decreases by one in the grabbing action, I guess you didn't notice the line because my code isn't formatted very well.